### PR TITLE
Add `and`,`or` and `not` conditions to the Compendium Browser trait filter

### PR DESCRIPTION
**Pull request by In3luki**
Recording 2023-02-18 at 18 06 57

### CODE REVIEW

**Comment by stwlam** [comment]
Maybe radios with "all of above" and "any of above"? For the "not" bit, how about a :no_entry_sign: next to the :x:?

**Comment by In3luki** [comment]
I tried a 🚫 element at first but I can't get it into the tag without breaking the styling of the whole thing. 😫

**Comment by In3luki** [comment]
image

**Comment by stwlam** [comment]
> image

I think we can eschew the "Condition" part.

**Comment by In3luki** [comment]
image

File: src/module/apps/compendium-browser/tabs/action.ts [hunk 1]
```diff
@@ -98,6 +95,7 @@ export class CompendiumBrowserActionTab extends CompendiumBrowserTab {
             },
             multiselects: {
                 traits: {
+                    condition: "and",
```
Comment (stwlam) [inline src/module/apps/compendium-browser/tabs/action.ts:98]:
```suggestion
                    conjunction: "and",
```

Comment (In3luki) [reply src/module/apps/compendium-browser/tabs/action.ts]: Done

**Pull request merged**